tracer/src/Datadog.Trace/Configuration/ImmutableAzureAppServiceSettings.cs

```diff
@@ -21,9 +21,24 @@ namespace Datadog.Trace.Configuration
     /// </summary>
     internal class ImmutableAzureAppServiceSettings
     {
-        private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor(typeof(ImmutableAzureAppServiceSettings));
+        private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor<ImmutableAzureAppServiceSettings>();
 
-        public static readonly string DefaultHttpClientExclusions = "logs.datadoghq, services.visualstudio, applicationinsights.azure, blob.core.windows.net/azure-webjobs, azurewebsites.net/admin, /azure-webjobs-hosts/".ToUpperInvariant();
+        /// <summary>
+        /// The http client paths we don't want to trace when running in AAS or Azure Functions.
+        /// </summary>
+        /// <seealso cref="Datadog.Trace.Configuration.TracerSettings.HttpClientExcludedUrlSubstrings"/>
+        public const string DefaultHttpClientExclusions =
+#pragma warning disable SA1025 // CodeMustNotContainMultipleWhitespaceInARow
+            ".LOGS.DATADOGHQ.," +
+            ".SERVICES.VISUALSTUDIO.," +
+            ".APPLICATIONINSIGHTS.AZURE.," +
+            ".MONITOR.AZURE.," +
+            ".BLOB.CORE.WINDOWS.NET/AZURE-WEBJOBS," +                   // no trailing slash, /azure-webjobs-hosts/*, /azure-webjobs-secrets/*, etc
+            ".TABLE.CORE.WINDOWS.NET/TABLES," +                         // no trailing slash, /Tables, /Tables('AzureFunctionsDiagnosticEventsCheck')
+            ".TABLE.CORE.WINDOWS.NET/AZUREFUNCTIONSDIAGNOSTICEVENTS," + // no trailing slash, /AzureFunctionsDiagnosticEvents202507()?$format=...
+            ".AZUREWEBSITES.NET/ADMIN/," +                              // trailing slash, /admin/*
+            "CDN.FUNCTIONS.AZURE.COM/PUBLIC/";                          // trailing slash, /public/*
```

Comment:
How sure are you that some of these won't be used by customers too? 🤔e.g. the CDN one looks like it could happen?

Reply:
I narrowed this one to `CDN.FUNCTIONS.AZURE.COM/PUBLIC/EXTENSIONBUNDLES/`.